assets/dev/scss/frontend/_swiper-customisations.scss

```diff
@@ -1,4 +1,5 @@
-.elementor-element {
+.elementor-element,
+.elementor-lightbox {
 
```

Comment:
remove blank line?

Reply:
@StuartElementor Done

Reply:
@louiswol94 Other reviewers like to have empty spaces between such elements. But I am not going to argue about it.